package.json

```diff
@@ -9,7 +9,7 @@
 		"upload",
 		"download"
 	],
-	"version": "2.4.0",
+	"version": "2.5.0",
```

Comment:
Setting to 2.5.0 so that npm publish step of release process will properly increment to 2.5.1.